src/main/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/JobView.java

```diff
@@ -28,20 +30,20 @@
         put(ABORTED,   "failing");  // if someone has aborted it then something is clearly not right, right? :)
     }};
 
-    public static JobView of(Job<?, ?> job) {
-        return new JobView(job, new BuildAugmentor(), RelativeLocation.of(job), new Date());
+    public static JobView of(Job<?, ?> job, Config config) {
```

Comment:
Could the most basic factory method `of()` take just the `Job<?, ?>` object, and not require the `Config` object to be passed in as well? 
This way we wouldn't have to repeat `withDefaultConfig()` in every test, only where it's necessary to express the intent to configure the view.